### PR TITLE
diff-pdf: add page

### PR DESCRIPTION
**Pull request by VicSegers**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 0.5

### CODE REVIEW

File: pages/common/diff-pdf.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# diff-pdf
+
+> Tool for visually comparing two PDFs.
+> More information: <https://github.com/vslavik/diff-pdf>.
+
+- Compare PDFs, (return code 0 = no differences, return code 1 = PDFs differ):
```
Comment (CleanMachine1) [inline pages/common/diff-pdf.md:6]: Is there no other indication?

```suggestion
- Compare PDFs using return codes (`0` = no difference, `1` = difference with PDFs)  :
```

File: pages/common/diff-pdf.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`diff-pdf a.pdf b.pdf`
```
Comment (CleanMachine1) [inline pages/common/diff-pdf.md:8]:
```suggestion
`diff-pdf {{path/to/a.pdf}} {{path/to/b.pdf}}`
```

Comment (CleanMachine1) [reply pages/common/diff-pdf.md:8]: Could you please apply this change throughout.

File: pages/common/diff-pdf.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Compare PDFs, output PDF with visually highlighted differences:
+
+`diff-pdf --output-diff=diff.pdf a.pdf b.pdf`
+
+- Compare PDFs, view differences in a simple GUI:
```
Comment (CleanMachine1) [inline pages/common/diff-pdf.md:14]:
```suggestion
- Compare PDFs, viewing differences in a simple GUI:
```

File: pages/common/diff-pdf.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# diff-pdf
+
+> Tool for visually comparing two PDFs.
+> More information: <https://github.com/vslavik/diff-pdf>.
+
+- Compare PDFs, (return code 0 = no differences, return code 1 = PDFs differ):
+
+`diff-pdf a.pdf b.pdf`
+
+- Compare PDFs, output PDF with visually highlighted differences:
```
Comment (CleanMachine1) [inline pages/common/diff-pdf.md:10]:
```suggestion
- Compare PDFs outputting a PDF with visually highlighted differences:
```

File: pages/common/diff-pdf.md (hunk 1 above)
Comment (VicSegers) [reply pages/common/diff-pdf.md:6]: Quoted from the creator: "It takes two PDF files as arguments. By default, its only output is its return code, which is 0 if there are no differences and 1 if the two PDFs differ."

Comment (CleanMachine1) [reply pages/common/diff-pdf.md:6]: Ok. I'll take their word :)

Whats your opinions on the suggestion?

File: pages/common/diff-pdf.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# diff-pdf
+
+> Tool for visually comparing two PDFs.
```
Comment (CleanMachine1) [inline pages/common/diff-pdf.md:3]:
```suggestion
> Tool for comparing two PDFs.
```

Considering that the base command doesn't do this, I think its best to keep it removed.

File: pages/common/diff-pdf.md (hunk 1 above)
Comment (VicSegers) [reply pages/common/diff-pdf.md:6]: Well, i agree that the brackets or comma should go, but it doesn't "use" return codes to compare the PDFs

Comment (CleanMachine1) [reply pages/common/diff-pdf.md:6]: Good point, my suggestion has been updated.

File: pages/common/diff-pdf.md (hunk 5 above)
Comment (VicSegers) [reply pages/common/diff-pdf.md:3]: Also quoted from the creator: "diff-pdf is a tool for visually comparing two PDFs.", but I agree your suggestion is better.

**Review by CleanMachine1 — APPROVED** [review]
LGTM thanks for your contribution!

**Review by blueskyson — APPROVED** [review]

**Pull request merged**